### ISSUE TITLE
FileNotFoundError for profiler.results

### ISSUE THREAD

**Issue by hollemantv** [spyder-ide/spyder#5107]
## Description

**What steps will reproduce the problem?**

1. Apologies in advance for the inadequecy of my response, but you guys asked me to submit this, so I'm doing so.  
2. I have no idea what happened. I hit F9 and a box popped up....no error message in console.  (fwiw i hit F9 to execute a 2-line function i wrote, with a range(3) for loop. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/lib/python3.6/site-packages/spyder_profiler/widgets/profilergui.py", line 260, in <lambda>
    self.finished(ec, es))
  File "/usr/lib/python3.6/site-packages/spyder_profiler/widgets/profilergui.py", line 325, in finished
    self.show_data(justanalyzed=True)
  File "/usr/lib/python3.6/site-packages/spyder_profiler/widgets/profilergui.py", line 346, in show_data
    self.datatree.load_data(self.DATAPATH)
  File "/usr/lib/python3.6/site-packages/spyder_profiler/widgets/profilergui.py", line 429, in load_data
    stats_indi = [pstats.Stats(profdatafile),]
  File "/usr/lib/python3.6/pstats.py", line 72, in __init__
    self.init(arg)
  File "/usr/lib/python3.6/pstats.py", line 86, in init
    self.load_stats(arg)
  File "/usr/lib/python3.6/pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/bch/.config/spyder-py3/profiler.results'

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.6.2
* Qt Versions: 5.9.1, PyQt5 5.9 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.2.2 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.7.2 (OK)

```

**Comment by ccordoba12** [comment]
What happens if you run

    spyder --reset

and try again?

**Comment by ReneSalhab** [comment]
I have the same problem. A reset didn´t help so far.

**Comment by ccordoba12** [comment]
This is a duplicate of issue #5587 and it was already fixed. The fix will be part of our 3.2.5 version.
